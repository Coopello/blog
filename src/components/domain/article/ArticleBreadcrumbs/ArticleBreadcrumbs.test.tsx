import renderer from "react-test-renderer";

import { ArticleBreadcrumbs } from "./";

describe("domain/article/ArticleBreadcrumbs", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <ArticleBreadcrumbs
        items={[{ label: "test", href: "/" }, { label: "test" }]}
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
