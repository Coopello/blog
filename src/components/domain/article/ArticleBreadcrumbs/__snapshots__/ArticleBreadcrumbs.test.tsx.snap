// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`domain/article/ArticleBreadcrumbs Snap Shot 1`] = `
<ul
  className="flex list-none flex-row gap-2 p-0 text-sm"
>
  <li
    className="m-0 p-0"
  >
    <a
      className="text-custom-primary underline hover:opacity-60"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      test
    </a>
  </li>
  <li
    className="m-0 p-0 text-custom-black"
  >
    <span>
      ï¼ž
    </span>
  </li>
  <li
    className="m-0 p-0 text-custom-black"
  >
    test
  </li>
</ul>
`;
